draft-pernet-mugnier-json-lines.md

```diff
@@ -19,10 +19,11 @@
 # Change the file extension to match the format (.xml for XML, etc...)
 #
 ###
-title: JSONLines Standard Content Type
+title: JavaScript Object Notation (JSON) Lines
+abbrev: JSON Lines
 category: info
 
-docname: draft-jsonlines-standard-content-type-latest
+docname: draft-pernet-mugnier-json-lines-latest

```

Comment:
It does look like it's not really recommended to have an hyphen: https://authors.ietf.org/en/naming-your-internet-draft

> Where possible, this component should not contain a hyphen. Hyphenated last names are allowed but having a hyphen in the second component makes it hard to extract programmatically and forces many tools to use heuristics when tying to separate the second component from the components that follow.

so simply `pernetmugnier` is good to me.